### PR TITLE
DigiByte Extension

### PR DESCRIPTION
**Pull request by nseidm1**
DigiByte Extension support.

### CODE REVIEW

**Comment by sidhujag** [comment]
please see #57

File: server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/digibyte/DigiByteExtension.java [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+            rate = new BigDecimal(st.nextToken());
+          } catch (Throwable e) {
+          }
+        }
+        String preferedFiatCurrency = ICurrencies.USD;
+        if (st.hasMoreTokens()) {
+          preferedFiatCurrency = st.nextToken().toUpperCase();
+        }
+        return new FixPriceRateSource(rate, preferedFiatCurrency);
+      } else if ("coinmarketcap".equalsIgnoreCase(exchangeType)) {
```
Comment (generalbytes) [inline server_extensions_extra/src/main/java/com/generalbytes/batm/server/extensions/extra/digibyte/DigiByteExtension.java]: Conflicting name with other ratesources

File: server_extensions_extra/src/main/resources/batm-extensions.xml [hunk 2]
```diff
@@ -227,6 +227,26 @@
...
+            <param name="host" />
+            <param name="port" />
+            <param name="accountname" />
+            <cryptocurrency>DGB</cryptocurrency>
+        </wallet>
+        <ratesource prefix="digibytefix" name ="Fix Price" >
+            <param name="price" />
+            <cryptocurrency>DGB</cryptocurrency>
+        </ratesource>
+        <ratesource prefix="coinmarketcap" name ="coinmarketcap.com" >
```
Comment (generalbytes) [inline server_extensions_extra/src/main/resources/batm-extensions.xml]: conflicting name

**Comment by nseidm1** [comment]
Livecoin rate source has replaced the duplicate coinmarketcap rate source.

**Comment by nseidm1** [comment]
Thank you for the quick PR feedback team, should be good to go now with a new recommended rate source from the DigiByte team.

**Comment by nseidm1** [comment]
All use of ICurrencies has been replaced with Currencies. Cheers.

**Comment by rpanak-generalbytes** [comment]
Thanks for the commits.

**Pull request merged**